_posts/2020-01-26-celebrate_1k_stars.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title:  "Celebrating the first 1k stars on GitHub"
+date:   2020-01-26
+categories: [jump-dev]
+permalink: /blog/1kstars/
+---
+
+January 26, 2020.
+
+# Celebrating the first 1k Stars of JuMP!
+
+JuMP just reached the 1k stars milestone on GitHub. JuMP started in 2012 as a experimental project and is now state-of-the-art modelling tool.
+
+Tracking the success of a open-source project is hard because we do no have access to the number of downloads, but we can check some metrics like the number of contributors, citations to code or papers describing the project.
+The number of citations of the two main JuMP papers are 253+428.
+Currently 71 developers have added some contribution in some of the 3747 commits of the JuMP repository made available in 67 releases. And this is only data from the main JuMP repo the ecosystem has many other repos ranging from MOI to solver wrappers.
+
+The number of stars of a GitHub project is yet another indicator, because it means people are adding the project to their list of favorites.
+
+Reaching these 1k stars is a very happy moment for the developers that can see some feedback on their work and motivates more work and new developments.
+
+
+<img src="/assets/news/1kstars/1kstars_plot.jpg" alt="The exponential growth of JuMP 2012 - 2020. Obtained with https://star-history.t9t.io/">
```

Comment:
The growth looks pretty linear since mid 2018. We don't need to claim that it's exponential.